### DISCUSSION TITLE
✅ 🔗 Modal Popup Widgets (Duplicate of #7106)

### DISCUSSION THREAD

**Discussion by rblackfox** [elementor/elementor#25238] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

There are only a few widgets that are 100% mandatory in a website build, the modal popup is one of this and is missing in Elementor

### Describe the solution you'd like.

Ultimate Add ons and other extensions have great modal popups to copy from:
https://ultimateelementor.com/widgets/modal-popup/

### Describe alternatives you've considered.

We use Ultimate Addons for Elementor, but development has essentially stopped on the extension, it would be nice to have this in Elementor Core. Thx!

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #7106

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.26.0! 🥳 

You can now use the **Off-Canvas** widget inside your **Loop Item** templates to display additional Dynamic Content for each post in your **Loop Grid** widgets! 🎉 🥳 🙌 

You can learn more about the Off-Canvas widget (and how to use it) here: 
https://elementor.com/help/off-canvas-widget/
And about the Loop Grid here: 
https://elementor.com/help/loop-grid/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
